tdrs-backend/tdpservice/parsers/schema_defs/ssp/m4.py

```diff
@@ -0,0 +1,146 @@
+"""Schema for SSP M1 record type."""
+
+
+from tdpservice.parsers.util import SchemaManager
+from tdpservice.parsers.fields import Field
+from tdpservice.parsers.row_schema import RowSchema
+from tdpservice.parsers import validators
+from tdpservice.search_indexes.models.ssp import SSP_M4
+
+m4 = SchemaManager(
+    schemas=[
+        RowSchema(
+            model=SSP_M4,
+            preparsing_validators=[
+                validators.hasLength(66),
+            ],
+            postparsing_validators=[],
+            fields=[
+                Field(
+                    item="0",
+                    name="RecordType",
+                    type="string",
+                    startIndex=0,
+                    endIndex=2,
+                    required=True,
+                    validators=[],
+                ),
+                Field(
+                    item="3",
+                    name="RPT_MONTH_YEAR",
+                    type="number",
+                    startIndex=2,
+                    endIndex=8,
+                    required=True,
+                    validators=[
+                        validators.dateYearIsLargerThan(1998),
+                        validators.dateMonthIsValid(),
+                    ],
+                ),
+                Field(
+                    item="5",
+                    name="CASE_NUMBER",
+                    type="string",
+                    startIndex=8,
+                    endIndex=19,
+                    required=True,
+                    validators=[validators.isAlphaNumeric()],
+                ),
+                Field(
+                    item="2",
+                    name="COUNTY_FIPS_CODE",
+                    type="string",
+                    startIndex=19,
+                    endIndex=22,
+                    required=True,
+                    validators=[validators.isInStringRange(0, 999)],
+                ),
+                Field(
+                    item="4",
+                    name="STRATUM",
+                    type="string",
+                    startIndex=22,
+                    endIndex=24,
+                    required=False,
+                    validators=[validators.isInStringRange(0, 99)],
+                ),
+                Field(
+                    item="6",
+                    name="ZIP_CODE",
+                    type="string",
+                    startIndex=24,
+                    endIndex=29,
+                    required=True,
+                    validators=[validators.isInStringRange(0, 99999)],
+                ),
+                Field(
+                    item="7",
+                    name="DISPOSITION",
+                    type="number",
+                    startIndex=29,
+                    endIndex=30,
+                    required=True,
+                    validators=[validators.isInLimits(1, 2)],
```

Comment:
```suggestion
                    validators=[validators.isInLimits(1)],
```

Reply:
we'll be making this change across all relevant record types with disposition. want to start flagging when stts submit disposition != 1